frontend/src/store/sc/actions.ts

```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
reducer & actions should be by page. move it to the `frontend/src/store/sc-configurate-instance/actions.ts`